src/jdk.sctp/unix/classes/sun/nio/ch/sctp/SctpChannelImpl.java

```diff
@@ -639,6 +639,7 @@ public void kill() throws IOException {
             if (state == ChannelState.KILLED)
                 return;
             if (state == ChannelState.UNINITIALIZED) {
+                SctpNet.close(fdVal);
                 state = ChannelState.KILLED;
```

Comment:
The change here seems mostly benign - there is no reason why either SctpChannel or SctpServerChannel would find themselves in this state - but it seems ok.